website/content/docs/k8s/connect/connect-ca-provider.mdx

```diff
@@ -89,27 +94,23 @@ $ kubectl create secret generic vault-config --from-file=config=vault-config.jso
 We will provide this secret and the Vault CA secret, to the Consul server via the
 `server.extraVolumes` Helm value.
 
-<CodeBlockConfig filename="config.yaml" highlight="4-13">
```

Comment:
My apologies for the late feedback. Is there reason this code block config was removed?

IMO specifying the filename on the code block was helpful, especially considering the next instruction is to install the chart using the file `config.yaml`.

Reply:
Ah that is my bad, I was doing a copy/paste and assumed that was some junk from github, woops!